development/image-syncer/external-images.yaml

```
@@ -44,8 +44,8 @@ images:
 - source: "quay.io/prometheus-operator/prometheus-operator:v0.58.0"
 - source: "quay.io/prometheus/alertmanager:v0.24.0"
 - source: "quay.io/prometheus/prometheus:v2.38.0"
-- source: "nats@sha256:93b6c178034659b624ec39316bec9daeb14b38f690a531eaf5687c8f9a6420f6"
-  tag: "2.9.2-alpine3.16"
+- source: "nats@sha256:0008930d6f49b1c3a706f5e751756e25faa913eb0d24a4d1a3cc7111995c2cc5"
```

https://hub.docker.com/layers/library/nats/2.9.3-alpine3.16/images/sha256-bf9cae35d4496a6557faaaa46222bc55961ffdea0f6d833968a7fe798581a289?context=explore
```suggestion
- source: "nats@sha256:bf9cae35d4496a6557faaaa46222bc55961ffdea0f6d833968a7fe798581a289"
```